client/state/user-settings/README.md

```diff
@@ -0,0 +1,23 @@
+User Settings
+=============
+
+This store holds user settings from `/me/settings`.
+
+# Usage
+
+1. Render `QueryUserSettings` from `components/data/query-user-settings`
+2. Connect your component specifying proper setting names:
+```js
+export default connect(
+	( state ) => {
+		return {
+			language: getUserSetting( state, 'language' ),
+		};
+	},
+	( dispatch ) => {
+		return bindActionCreators( {
+			saveLanguage: language => saveUserSettings( { language } ),
+		}, dispatch );
+	},
```

Comment:
these are fine examples but also fairly verbose, maybe using arrow-function syntax for `connect()` could bring out the important details a bit more

```js
export default connect(
	state => ( { language: getUserSettings( state, 'language' ) } ),
	dispatch => ( { saveLanguage: language => dispatch( saveUserSettings( { language } ) ) } ),
)( Account );
```